stories/LinkComponent-stories.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Echoes React
+ * Copyright (C) 2023-2023 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { Link, LinkSubdued as LinkSubduedComp } from '@sonarsource/echoes-react';
+import type { Meta, StoryFn, StoryObj } from '@storybook/react';
+import { OpenNewTabIcon } from '../src/common/icons/OpenNewTabIcon';
+
+const meta: Meta<typeof Link> = {
+  component: Link,
+  title: 'Links',
+  args: {
+    to: '/path',
+  },
+} satisfies Meta<typeof Link>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const LinkDefault: Story = {
+  args: {
+    children: 'Link Default',
+    to: '/path',
+  },
+};
+
```

Comment:
How about stories with some context:

- A link inside a paragraph of text
- A list of links (`ul`)
- a link and other things in a flex-row div with align-items-center

just random ideas, to make sure it behaves nicely in a layout